2025-04-28 15:58:51,992 INFO    MainThread:319 [wandb_init.py:setup_run_log_directory():647] Logging user logs to /home/jlai/ML/wandb/run-20250428_155851-o9athga1/logs/debug.log
2025-04-28 15:58:51,992 INFO    MainThread:319 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to /home/jlai/ML/wandb/run-20250428_155851-o9athga1/logs/debug-internal.log
2025-04-28 15:58:51,992 INFO    MainThread:319 [wandb_init.py:monkeypatch_ipython():599] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f464f355340>
2025-04-28 15:58:51,993 INFO    MainThread:319 [wandb_init.py:init():761] calling init triggers
2025-04-28 15:58:51,993 INFO    MainThread:319 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-04-28 15:58:51,993 INFO    MainThread:319 [wandb_init.py:init():784] starting backend
2025-04-28 15:58:51,993 INFO    MainThread:319 [wandb_init.py:init():788] sending inform_init request
2025-04-28 15:58:51,994 INFO    MainThread:319 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-04-28 15:58:51,994 INFO    MainThread:319 [wandb_init.py:init():803] backend started and connected
2025-04-28 15:58:52,000 INFO    MainThread:319 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2025-04-28 15:58:52,000 INFO    MainThread:319 [wandb_run.py:_label_probe_notebook():1214] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2025-04-28 15:58:52,000 INFO    MainThread:319 [wandb_init.py:init():896] updated telemetry
2025-04-28 15:58:52,001 INFO    MainThread:319 [wandb_init.py:init():920] communicating run to backend with 90.0 second timeout
2025-04-28 15:58:52,395 INFO    MainThread:319 [wandb_init.py:init():995] starting run threads in backend
2025-04-28 15:58:52,547 INFO    MainThread:319 [wandb_run.py:_console_start():2377] atexit reg
2025-04-28 15:58:52,548 INFO    MainThread:319 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-04-28 15:58:52,548 INFO    MainThread:319 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-04-28 15:58:52,548 INFO    MainThread:319 [wandb_run.py:_redirect():2317] Redirects installed.
2025-04-28 15:58:52,549 INFO    MainThread:319 [wandb_init.py:init():1037] run started, returning control to user process
2025-04-28 16:01:34,026 INFO    MainThread:319 [wandb_run.py:_config_callback():1261] config_cb None None {'gamma': 0.400541219604713, 'learning_rate': 0.09582846948810853, 'max_depth': 6, 'n_estimators': 261, 'subsample': 0.945557438533904}
2025-04-28 16:01:37,907 INFO    MainThread:319 [wandb_run.py:_finish():2112] finishing run dlai211/Dark_ph BDT Random/o9athga1
2025-04-28 16:01:37,908 INFO    MainThread:319 [jupyter.py:save_history():458] not saving jupyter history
2025-04-28 16:01:37,908 INFO    MainThread:319 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-04-28 16:01:37,908 INFO    MainThread:319 [wandb_init.py:_jupyter_teardown():581] cleaning up jupyter logic
2025-04-28 16:01:37,908 INFO    MainThread:319 [wandb_run.py:_atexit_cleanup():2342] got exitcode: 0
2025-04-28 16:01:37,908 INFO    MainThread:319 [wandb_run.py:_restore():2324] restore
2025-04-28 16:01:37,908 INFO    MainThread:319 [wandb_run.py:_restore():2330] restore done
2025-04-28 16:01:39,434 INFO    MsgRouterThr:319 [mailbox.py:close():115] Closing mailbox, abandoning 1 handles.
2025-04-28 16:01:39,435 INFO    MainThread:319 [wandb_run.py:_footer_history_summary_info():3958] rendering history
2025-04-28 16:01:39,435 INFO    MainThread:319 [wandb_run.py:_footer_history_summary_info():3990] rendering summary
2025-04-28 16:01:39,437 INFO    MainThread:319 [wandb_run.py:_footer_sync_info():3919] logging synced files
